lib/src/definitions.dart

```diff
@@ -304,13 +304,16 @@ class OnSpeakAloudTextResult {
   /// A [String] that represents the language code, i.e. es-ES
   final String? lang;
 
-  /// A [Double] that represents the volume from 0.0 to 1.0
+  /// A [Double] that represents the volume from 0.0 to 1.0, default 1
```

Comment:
Eu prefería que aparecera aquí especificado o que é menos, máis, lento, rápido, etc. Pero de todas formas o enlace á documentación é útil. 

Ahora ben, ollo porque en dart docs os enlaces van entre corchetes:

/// Para máis información, consulta [Google](https://www.google.com).